demo/holdings.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function
+from datetime import datetime, timedelta
+import datetime
+import csv
+import swagger_client
+from demo import configuration, authed_client
+
+# declare an instance of the api for interacting with the account holdings resource
+api = swagger_client.AccountHoldingsApi(authed_client())
+
+
+def list_latest_holdings():
+    """
+    Demonstrates listing API keys
+    API keys can be managed (created, read, deleted) by creating a temporary oauth2 id token
```

Comment:
This comment should be updated to refer to this method it is still referring to the API key demo